sql-movies/test/06-simple-queries.test.ts

```diff
@@ -27,7 +33,12 @@ describe("Simple Queries", () => {
   it(
     "should select count from movies where budget was more than 100000000 and release date after 2009",
     async done => {
-      const query = `todo`;
+      const query = `
+        SELECT 
+          COUNT(*) AS count
+        FROM ${MOVIES}
+        WHERE budget > 100000000
```

Comment:
prasīts 2009 + gads. :) te redzu tikai general vaicājumu ne par laika ierobežojumu. :) šis nebūs pareizi.
un iespējams ka bija arī budget_adjusted colonna jālieto. :)